apps/browser/settings/declarativeloginmodel.h

```diff
@@ -0,0 +1,61 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef LOGINMODEL_H
+#define LOGINMODEL_H
+
+#include <QAbstractListModel>
+#include <QQmlParserStatus>
+
+#include "logininfo.h"
+
+class DeclarativeLoginModel : public QAbstractListModel, public QQmlParserStatus
+{
+    Q_OBJECT
+    Q_INTERFACES(QQmlParserStatus)
+
+    Q_PROPERTY(int count READ count NOTIFY countChanged)
```

Comment:
```suggestion
    Q_PROPERTY(int count READ rowCount NOTIFY countChanged)
```
I'm not sure if it's ok for you, but you can use 'rowCount()' instead additional function

Reply:
Thanks, nice idea. I've made this change.